ui/.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
@iatdaitan can you issue the following on your repository's root:
`git ignore ui/.vscode; git rm --cached -r ui/.vscode; git add .gitignore;`
and then commit it and push it?